src/modules/item/actions.ts

```diff
@@ -64,6 +64,32 @@ export type SaveItemRequestAction = ReturnType<typeof saveItemRequest>
 export type SaveItemSuccessAction = ReturnType<typeof saveItemSuccess>
 export type SaveItemFailureAction = ReturnType<typeof saveItemFailure>
 
+// Save multiple items
+
+export const SAVE_MULTIPLE_ITEMS_REQUEST = '[Request] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_SUCCESS = '[Success] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_FAILURE = '[Failure] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CANCELLED = '[Cancelled] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CANCEL = '[Cancel] Save Multiple Items'
+export const SAVE_MULTIPLE_ITEMS_CLEAR_STATE = '[Clear State] Save Multiple Items'
+
+export const saveMultipleItemsRequest = (builtFiles: BuiltFile<Blob>[]) => action(SAVE_MULTIPLE_ITEMS_REQUEST, { builtFiles })
+export const saveMultipleItemsFailure = (error: string, items: Item[], fileNames: string[]) =>
+  action(SAVE_MULTIPLE_ITEMS_FAILURE, { error, items, fileNames })
+export const saveMultipleItemsCancelled = (items: Item[], fileNames: string[]) =>
+  action(SAVE_MULTIPLE_ITEMS_CANCELLED, { items, fileNames })
+export const saveMultipleItemsSuccess = (items: Item[], fileNames: string[]) => action(SAVE_MULTIPLE_ITEMS_SUCCESS, { items, fileNames })
+
+export const cancelSaveMultipleItems = () => action(SAVE_MULTIPLE_ITEMS_CANCEL)
+export const clearStateSaveMultipleItems = () => action(SAVE_MULTIPLE_ITEMS_CLEAR_STATE)
```

Comment:
following the comment above, the first action is well named, the second could maybe drop `State`